zero-module.yml

```diff
@@ -242,3 +252,25 @@ conditions:
     data:
     - src/app/billing
     - scripts/stripe-example-setup.sh
+  - action: ignoreFile
+    matchField: serverless
+    whenValue: "yes"
+    data:
+    - kubernetes
+    - .github/workflows/ci.yml
+  - action: ignoreFile
+    matchField: serverless
+    whenValue: "no"
+    data:
+    - auth
+    - .github/workflows/sam.yml
```

Comment:
We probably don't want to exclude files here since that would make it harder to do the upgrade from SAM -> EKS, but instead leave the files and instead have a way to disable the functionality until necessary with flags in the code.

Reply:
fixed, only ignoring files in EKS case